### DISCUSSION TITLE
What is your favourite game?

### DISCUSSION THREAD

**Discussion by H-S-0811** [NAIST-SE/AttractingContributors#1188] [2024 Attraction Contributions]
https://github.com/H-S-0811/AttractingContributors

**Comment by marshr00m** [comment]
I submitted a pull request. I would appreciate any additions to my list!
https://github.com/marshr00m/AttractingContributors-Where-would-you-like-to-travel

**Comment by masataka-ki** [comment]
I've sent you a pull request, so if you want, please pull request for me too!
https://github.com/masataka-ki/AttractingContributors-Please-tell-me-a-country-you-want-to-visit-in-the-future
